src/widget/OktaSignIn.js

```diff
@@ -130,10 +130,18 @@ var OktaSignIn = (function () {
       .fail(error);
     }
 
+    /**
+     * Returns the Okta Auth JS auth client.
+     */
+    function getAuthClient() {
+      return authClient;
```

Comment:
My thinking could be off here, so I'd appreciate some feedback. 

I wanted to keep this as a `getter` so we could easily modify the `authClient` object as needed. For example, if we decide to omit the `fingerPrint` method, we _could_ easily transform the `authClient` before returning it.

Reply:
Technically you can use [property getters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) to solve the same problem.  If we feel like we want to hide auth JS functionality in the future we should be having a bigger conversation about why we're doing this in the first place.

Reply:
Property getters would be great if this library already used that design, so I'm hesitant on updating this PR to reflect that.

Regarding hiding AuthJS functionality - I'm not sure I see a valid case **today** for needing to do so, but pulling it out into a `getter` allows us to in the future.